website/views/shared/modals/members.jade

```diff
@@ -62,14 +62,14 @@ script(type='text/ng-template', id='modals/private-message.html')
 
 script(type='text/ng-template', id='modals/send-gift.html')
   .modal-header
-    h4 Send Gift to {{::profile.profile.name}}
+    h4=env.t('sendGiftHeading', {name: "{{profile.profile.name}}"})
```

Comment:
I noticed some other places that could use some one time bindings, so I just added them in: https://github.com/HabitRPG/habitrpg/commit/fce9d39f19cb3080e6bca4e1b96e5a7d9ca20ee6